docs/comparisons.md

```diff
@@ -0,0 +1,41 @@
+# Signing Comparisons
+
+## NuGet
+
+The following tables summarize differences between NuGet, dotnet, and Sign CLI's. 
+
+### Features
+
+Feature | NuGet CLI | dotnet CLI | Sign CLI
+-- | -- | -- | --
+Use signing certificate from the file system | ✔️ | ✔️ | ❌
+Use signing certificate from a local store | ✔️ | ✔️ | ❌
+Use signing certificate from Azure Key Vault | ❌ | ❌ | ✔️
+Identify signing certificate by fingerprint | ✔️ | ✔️ | ❌
+Identify signing certificate by subject name | ✔️ | ✔️ | ❌
+Identify signing certificate by name (user-defined) | ❌ | ❌ | ✔️
+Can skip timestamping | ✔️ | ✔️ | ❌
+Opt-in required to overwrite already signed package | ✔️ | ✔️ | ❌
+Can sign files (e.g.: *.dll) inside package | ❌ | ❌ | ✔️
+Can verify signed package | ✔️ | ✔️ | ❌
+
+### Platform support
+Platform | NuGet CLI | dotnet CLI | Sign CLI
+-- | -- | -- | --
+Windows x86 | ✔️ | ✔️ | ❌
+Windows x64 | ✔️ | ✔️ | ✔️
+Windows ARM64 | ❌ | ✔️ | ❌
+Linux | ❌ | ✔️ | ❌
+macOS | ❌ | ✔️ | ❌
+
+### Requirements
+
+Requirement | NuGet CLI | dotnet CLI | Sign CLI
+-- | -- | -- | --
+.NET Framework | ✔️ (>= 4.7.2) | ❌ | ❌
+.NET SDK | ❌ | ✔️ (>= 7) | ❌
```

Comment:
Technically `dotnet cli` can sign packages starting with `5.x.x`, right?

Reply:
It's more complicated than that.  I was trying to keep it simple.  I'll update with more detail.